babel-macro/package.json

```diff
@@ -7,7 +7,7 @@
   ],
   "description": "",
   "scripts": {
-    "build": "rm -rf dist && tsc && cp -R 'src/test/backend/' 'dist/test/backend'",
+    "build": "rm -rf dist && tsc && cp -r src/test/backend/ dist/test/",
```

Comment:
From `man cp`
```
COMPATIBILITY
	Historic versions of the cp utility had a -r option.
	This implementation supports that option; however, its use is strongly discouraged, as it does not correctly copy special files, symbolic links, or fifo's.
```

Reply:
It's not in the linux man page http://man7.org/linux/man-pages/man1/cp.1.html

Reply:
So you're saying that you *want to stop* supporting development on mac? (as you are specifically changing from the cross-platform option to the linux only option and this is the only actual change that is happening here)

I don't want to block, so I'm approving but from now on I'll take under-consideration that building on our machines is not necessary nor supporting developers who wants to contribute but don't know how to use docker...

Reply:
I think the changes between BSD and Linux can be ignored, and something like `cp -a` would be unfamiliar.

Reply:
Just note that this still works on Mac..

Reply:
Current state: using -R
Proposed change: using -r
difference in linux - none, they both work the same
difference in osx - going from recommended to deprected
difference in windows - none, they both don't work

So this change as I see it, is only to stick it to apple.
And it's an explicit change just for that, it causes no other change.